GGO.Singleplayer/Main.cs

```diff
@@ -75,6 +79,16 @@ private void OnTick(object Sender, EventArgs Args)
                         // To end this up, increase the count of peds "rendered"
                         Count++;
                     }
+
+                    // Check for on screen dead Peds to display dead markers, limit to 10 to not clutter the screen.
+                    if (NearbyPed.IsDead && NearbyPed.IsOnScreen && DeadCount < 10)

```

Comment:
```suggestion
                    if (NearbyPed.IsDead && NearbyPed.IsOnScreen)
```
See the picture on #7, we want to show all of them.